app/src/components/attachments/DropZone.tsx

```diff
@@ -75,7 +75,7 @@ export const DropZone: React.FC<IDropZoneProps & IDropZoneConfigProps> = (props)
               <Typography variant="h3" className={classes.textSpacing}>
                 Drag your files here, or <Link className={classes.browseLink}>Browse Files</Link>
               </Typography>
-              {!!maxFileSize && maxFileSize !== Infinity && (
+              {!!maxFileSize && maxFileSize !== Infinity && acceptedFileExtensions && (
```

Comment:
I don't think we need the maxFileSize check here.  It won't impact the extensions we allow